src/plugin.js

```diff
@@ -55,7 +56,7 @@ class Flvjs extends Html5 {
  */
 Flvjs.isSupported = function() {
 
-  return window.flvjs && window.flvjs.isSupported();
```

Comment:
We don't need to check for `flvjs` if we're importing it.

Reply:
done